{{cookiecutter.project_slug}}/.claude/agents/test-engineer.md

```
@@ -0,0 +1,79 @@
+
+# Run specific category
+uv run pytest -m "unit"
+uv run pytest -m "integration"
+
+# Run mutation testing
+uv run mutmut run
+```
+
+## Invocation
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)